src/Entity/ApiDoc.php

```diff
@@ -214,24 +260,38 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
       ->setDisplayConfigurable('form', TRUE)
       ->setDisplayConfigurable('view', TRUE);
 
-    $fields['status'] = BaseFieldDefinition::create('boolean')
```

Comment:
Shouldn't `BaseFieldDefinition::create('boolean')` be left on this?

Reply:
There is a new parent class that creates the status field. See: `\Drupal\Core\Entity\EditorialContentEntityBase::baseFieldDefinitions()`.